src/events/tab_event_handler.ts

```diff
@@ -42,22 +46,31 @@ export class tabEventHandler {
 		const configUtils = new ConfigUtils();
 		const repoId = configUtils.getRepoIdByPath(this.repoPath);
 		if (!repoId) return
-
+		
 		// Get list of current tabs
 		const open_tabs = vscode.window.tabGroups.all;
-		const tabs: ITabFile[] = [] = open_tabs.flatMap(tab_group => 
+
+		const tabs: ITabFile[] = open_tabs.flatMap(tab_group => 
 			tab_group.tabs.map(tab => {
 				// Get path of tab
 				// @ts-ignore
-				const fileRelativePath = tab.input.uri.path;
+				const tabFilePath = tab.input.uri.path;
+				const splitPath = tabFilePath.split(`${this.repoPath}${path.sep}`);
+				if (splitPath.length == 1) {
+					CodeSyncLogger.info(`Removing file from other repo: Skipping invalid tab: ${tabFilePath}`);
```

Comment:
Log statement seems incorrect, we are not deleting anything here, just ignoring tabs of other repos, right?

Reply:
That is correct, I am updating it